syft_tensorflow/hook/hook.py

```diff
@@ -21,13 +21,15 @@ def __init__(
         local_worker: BaseWorker = None,
         is_client: bool = True
     ):
-        self.tensorflow = tensorflow
-        self.tensorflow.hook = self
+
+        self.tensorflow = tf
```

Comment:
We want to assign the module that's passed in via the tensorflow arg and not the tf module that's imported above, so this needs to be fixed.